rotkehlchen/api/rest.py

```diff
@@ -4723,9 +4764,27 @@ def export_history_events(
                 headers.keys(),
             )
         except (CSVWriteError, PermissionError) as e:
-            return api_response(wrap_in_fail_result(str(e)), status_code=HTTPStatus.CONFLICT)
+            return wrap_in_fail_result(message=str(e), status_code=HTTPStatus.CONFLICT)
 
-        return api_response(OK_RESULT, status_code=HTTPStatus.OK)
+        return OK_RESULT
+
+    @staticmethod
+    def get_rate_in_profit_currency(asset: Asset, timestamp: Timestamp, currency: Asset) -> Price:
+        """Get the profit_currency price of asset in the given timestamp
++
+        May raise:
+        - PriceQueryUnsupportedAsset if from/to asset is missing from price oracles
+        - NoPriceForGivenTimestamp if we can't find a price for the asset in the given
+        timestamp from the price oracle
+        - RemoteError if there is a problem reaching the price oracle server
+        or with reading the response returned by the server
+        """
+        rate = PriceHistorian.query_historical_price(
+            from_asset=asset,
+            to_asset=currency,
+            timestamp=timestamp,
+        )
+        return rate
```

Comment:
why did you keep it? We said about removing it. Now is just a wrapper around `query_historical_price`